modules/signals/rxjs-interop/src/rx-method.ts

```diff
@@ -11,7 +11,8 @@ import {
 import { isObservable, noop, Observable, Subject, Unsubscribable } from 'rxjs';
 
 type RxMethod<Input> = ((
-  input: Input | Signal<Input> | Observable<Input>
+  input: Input | Signal<Input> | Observable<Input>,
+  injector?: Injector
```

Comment:
for consistency with the `rxMethod` signature, I suggest:

```suggestion
  config?: { injector?: Injector }
```